BasicClasses/BasicClasses.tex

```diff
@@ -620,9 +609,10 @@ \section{Strings}
 
 \noindent
 Another important difference is that a \ct{String} is mutable, whereas a \ct{Symbol} is immutable.
+Still, we need to copy a string constant before altering it because constants are stored as literals in the compiled method and code literals are read-only:
```

Comment:
hm... too detailed at this point in the book. If you find a way to frame it without requiring readers to grasp the notion of something stored in a method, it might work.

Reply:
Is this better? fff789d7e86f778de428e18c7ca6d402b8532944